src/components/UserAccess/UserAccessListView.tsx

```diff
@@ -71,10 +71,12 @@ export const UserAccessListView: React.FC<React.PropsWithChildren<unknown>> = ()
 
   const filterRBs = React.useMemo(
     () =>
-      roleBindings.filter((rb) =>
-        rb.subjects.some((subject) =>
-          subject.name.toLowerCase().includes(usernameFilter.toLowerCase()),
-        ),
+      roleBindings.filter(
+        (rb) =>
+          !rb.subjects ||
```

Comment:
Should we just filter out the rolebindings and not show it on the UI if it doesn't contain any subjects?